config/example.config.js

```diff
@@ -0,0 +1,19 @@
+import pageRoutes from './router.config';
+
+export default {
+  // Note: typeof(process.env.MOCK) === string
+  define: {
+    MOCK_UI: process.env.MOCK === 'true',
+    // Use http://localhost:8000/api/v1/endpoints for development environment
+    // to reference the mock dev server.
+    ENDPOINT: 'PBENCH_SERVER_ENDPOINT',
```

Comment:
I don't understand the intent of this comment. In real production we want to fetch from `/api/v1/endpoints` to implicitly use `window.origin` host; for your "hosted developer mode" you want to point to a real server like `http://pbench-server.example.com/api/v1/endpoints`; you don't ever really want to point to your local npm server. In the local e2e, we *mock* the `/api/v1/endpoints` call entirely without changing the `fetch` target.

Reply:
This file needs to go from this PR, the development configuration is `config/config.js` by default. For prod, you can use `config/config.prod.js`